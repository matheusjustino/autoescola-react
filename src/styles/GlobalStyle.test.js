import React from 'react';
import { render } from '@testing-library/react';

import GlobalStyle from './GlobalStyle';

test('match snapshot', () => {
	const { baseElement } = render(<GlobalStyle/>);

	expect(baseElement).toMatchSnapshot();
});
