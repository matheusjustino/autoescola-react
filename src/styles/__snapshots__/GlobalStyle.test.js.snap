// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
@media (min-width:992px) {

}

@media (min-width:992px) {

}

@media (min-width:992px) {

}

@media (min-width:992px) {

}

@media (min-width:992px) {

}

@media (min-width:992px) {

}

<body>
  <div />
</body>
`;
